### PR TITLE
Add examples/charge_offpeak.py script

### PR DESCRIPTION
**Pull request by hbergh**

### CODE REVIEW

File: examples/charge_offpeak.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    dlat = math.radians(lat2 - lat1)
+    dlon = math.radians(lon2 - lon1)
+    a = (math.sin(dlat / 2) * math.sin(dlat / 2) +
+         math.cos(math.radians(lat1)) * math.cos(math.radians(lat2)) *
+         math.sin(dlon / 2) * math.sin(dlon / 2))
+    c = 2 * math.atan2(math.sqrt(a), math.sqrt(1 - a))
+    d = radius * c
+
+    return d
+
```
Comment (ardevd) [inline examples/charge_offpeak.py]: Too many linebreaks here.

**Comment by hbergh** [comment]
Okay. It was a straight copy/paste of code I found which I didn't edit for style.  Will remove them.

File: examples/charge_offpeak.py (hunk 1 above)
Comment (hbergh) [reply examples/charge_offpeak.py]: Okay. It was a straight copy/paste of code I found which I didn't edit for style. Will remove them.

**Comment by ardevd** [comment]
Looks great. Thank you very much!

**Pull request merged**